index.html

```diff
@@ -0,0 +1,170 @@
+            <li><a href=#><img src='./images/twitter.svg' alt="twitter icon"></a></li>
+            <li><a href=#><img src='./images/Linkedin_icon.svg' alt="LinkedIn icon"></a></li>
+            <li><a href=#><img src='./images/m_icon.svg' alt="M icon"></a></li>
+            <li><a href=#><img src='./images/cat_icon.svg' alt="Cat icon"></a></li>
+            <li><a href=#><img src='./images/hand_icon.svg' alt="Hand icon"></a></li>
+        </ul>
+    </section>
+    <div class="grid-container">
+        <div class="grid-item item1">
+            <img src='./images/portfolio1.svg' alt="portfolio image">
+            <h1>Tonic</h1>
+            <div>
+                <h4>CANOPY</h4>
+                <ul>
+                    <li><img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;Back End Dev</li>
+                    <li>&nbsp;&nbsp;<img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;2015</li>
+                </ul>
+            </div>
+            <p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
+            <ul class="diff_classes">
+                <li><h4>html</h4></li>
+                <li><h4>css</h4></li>
+                <li><h4>javaScript</h4></li>
+            </ul>
+            <h3>See Project</h3>
+        </div>
+        <div class="grid-item item2">
+            <img src='./images/portfolio2.svg' alt="portfolio image">
+            <h1>Tonic</h1>
+            <div>
+                <h4>CANOPY</h4>
+                <ul>
+                    <li><img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;Back End Dev</li>
+                    <li>&nbsp;&nbsp;<img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;2015</li>
+                </ul>
+            </div>
+            <p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
+            <ul class="diff_classes">
+                <li><h4>html</h4></li>
+                <li><h4>css</h4></li>
+                <li><h4>javaScript</h4></li>
+            </ul>
+            <h3>See Project</h3>
+        </div>
+        <div class="grid-item item3">
+            <img src='./images/portfolio3.svg' alt="portfolio image">
+            <h1>Tonic</h1>
+            <div>
+                <h4>CANOPY</h4>
+                <ul>
+                    <li><img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;Back End Dev</li>
+                    <li>&nbsp;&nbsp;<img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;2015</li>
+                </ul>
+            </div>
+            <p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
+            <ul class="diff_classes">
+                <li><h4>html</h4></li>
```

Comment:
- Fix the requested design on your Figma template:
![Screen Shot 2021-09-25 at 17 38 10](https://user-images.githubusercontent.com/22258274/134787712-c2bb7cd7-5ca6-4f1e-af38-6a499ebc3e54.png)
#
- Yours:
![Screen Shot 2021-09-25 at 17 39 30](https://user-images.githubusercontent.com/22258274/134787716-c0a6019e-3503-4c66-82bb-072d7d162c76.png)

Reply:
Fixed this issue.